data/templates/css/wikihow.scss

```diff
@@ -360,3 +360,31 @@ h2 + ol, h3 + ol, .steps_list_2 {
         }
     }
 }
+
+@media (max-width: 360px) {
```

Comment:
Can you use `device-width` so that we don't unexpectedly get the mobile style when users size down an app window on the desktop?

Reply:
That works too indeed.